modules/oc-mirror-procedure-delete-v2.adoc

```diff
@@ -38,18 +38,15 @@ delete:
          minVersion: <operator_max_version> <5>
          maxVersion: <operator_min_version> <5>
   additionalImages:
-    - name: <additional_images> <6>
+    - name: <additional_images>
 ----
 <1> Specify the name of the {product-title} channel to delete, for example `stable-4.15`.
 <2> Specify a version range of the images to delete within the channel, for example `4.15.0` for the minimum version and `4.15.1` for the maximum version.
 To delete only one version's images, use that version number for both the `minVersion` and `maxVersion` fields.
-<3> Specify an Operator catalog image to delete, for example `registry.redhat.io/redhat/redhat-operator-index:v4.14`.
-To delete the catalog image, you must not specify individual Operators using the `delete.operators.packages` parameter.
-<4> Specify a specific Operator bundle to delete, for example `aws-load-balancer-operator`.
-If you specify individual Operators, the Operator catalog image does not get deleted.
+<3> Specify an Operator catalog image containing the operators you'd like to delete, for example `registry.redhat.io/redhat/redhat-operator-index:v4.14`.
```

Comment:
Capitalize Operators, also I think you can delete "you'd like" since we can just be more assertive here:
```suggestion
<3> Specify an Operator catalog image containing the Operators to delete, for example `registry.redhat.io/redhat/redhat-operator-index:v4.14`.
```